core/types/l2trace.go

```diff
@@ -191,3 +191,12 @@ func NewTransactionData(tx *Transaction, blockNumber uint64, config *params.Chai
 	}
 	return result
 }
+
+// WrapProof turn the bytes array into proof type (array of hexutil.Bytes)
+func WrapProof(proofBytes [][]byte) (wrappedProof []hexutil.Bytes) {
```

Comment:
we can refactor the tracing codes using this `WrapProof` 
after this PR being merged.
currently I am not doing it because I want to reduce the changes of tracing codes and make it easier to review